### PR TITLE
Add privacy practices detail screen

### PR DESCRIPTION
**Pull request by subsymbolic**
Asana Issue URL: https://app.asana.com/0/488551667048375/471193197990042

## Description
Displays privacy practices details

## Steps to Test this PR:
**TEST ONE:**
1. Visit google.com
1. Open the Privacy Dashboard and tap open Privacy Practices
1. Practices should show as poor and poor practices should be shown

**TEST TWO:**
1. Visit duckduckgo.com
1. Open the Privacy Dashboard and tap open Privacy Practices
1. Practices should show as good and good practice should be shown

**TEST THREE:**
1. Visit netflix.com
1. Open the Privacy Dashboard and tap open Privacy Practices
1. Practices should show as mixed and mixed practices should be shown

**TEST FOUR:**
1. Visit cnn.com
1. Open the Privacy Dashboard and tap open Privacy Practices
1. Practices should show as unknown and no practices should be shown (we will later add an additional message in the blank space)

**TEST FIVE**
Following from the above test, tap the TOSD panel at the bottom of the screen and ensure that it launches the tosdr url

### CODE REVIEW

File: app/src/androidTest/java/com/duckduckgo/app/privacymonitor/ui/PrivacyPracticesViewModelTest.kt [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    @get:Rule
+    @Suppress("unused")
+    var instantTaskExecutorRule = InstantTaskExecutorRule()
+
+    private var viewStateObserver: Observer<PrivacyPracticesViewModel.ViewState> = mock()
+
+    private val testee: PrivacyPracticesViewModel by lazy {
+        val model = PrivacyPracticesViewModel()
+        model.viewState.observeForever(viewStateObserver)
+        model
```
Comment (CDRussell) [inline app/src/androidTest/java/com/duckduckgo/app/privacymonitor/ui/PrivacyPracticesViewModelTest.kt]: ?

**Comment by CDRussell** [comment]
Looks good, other than one oddity above @subsymbolic

File: app/src/androidTest/java/com/duckduckgo/app/privacymonitor/ui/PrivacyPracticesViewModelTest.kt (hunk 1 above)
Comment (subsymbolic) [reply app/src/androidTest/java/com/duckduckgo/app/privacymonitor/ui/PrivacyPracticesViewModelTest.kt]: Haha looks a bit weird right? It's a pattern I'm getting into for tests. The lazy load lets any all the mocking get setup before the model is generated. It's useful but if you find it really offensive I can refactor it out by generating the model in each test method

Comment (CDRussell) [reply app/src/androidTest/java/com/duckduckgo/app/privacymonitor/ui/PrivacyPracticesViewModelTest.kt]: It _looked_ weird, but i get it now. the stray `model` threw me as i was forgetting about implicit returns.

no action needed. 👍

**Comment by subsymbolic** [comment]
Thanks for the review @CDRussell

**Pull request merged**